src/apps/service/oauth.clj

```diff
@@ -69,15 +69,25 @@
 (defn authorization-uri
   "Generates an authorization URI for a remote API."
   [{:keys [api-name] :as server-info} username state-info]
-  (when-not (op/get-access-token api-name username)
```

Comment:
Ah, it must be a Friday afternoon...
So `authorization-uri` no longer does a `when-not` check so it will no longer return nil 💡